src/containers/gui.jsx

```diff
@@ -102,6 +139,17 @@ class GUI extends React.Component {
             loadingStateVisible,
             ...componentProps
         } = this.props;
+
+        // TODO: Determine if we finished loading previously. In which case we
+        // should render the GUI anyways with a loading screen here or in the
+        // GUI. Otherwise we will unload the GUI when we switch projects (such
+        // as with "Load from your computer").
+        if (!fontsLoaded || fetchingProject || isLoading) {
+            // TODO: Render a loading screen.
+            return null;
```

Comment:
Cool.

This will cause a screen flash. When the <Loader /> is replaced by the Loader in GUI the animation will restart because it been a new DOM element. Fixing this flash will be a next step perhaps.

Reply:
Or. Hmm. I'm not sure that will happen. It will probably normally not happen since that be the same render when loading is false.

Uhm.